examples/react-todomvc/server.js

```diff
@@ -1,26 +1,40 @@
-var webpack = require('webpack');
-var WebpackDevServer = require('webpack-dev-server');
-var openInEditor = require('express-open-in-editor');
-var sourceFragment = require('express-source-fragment');
-var config = require('./webpack.config');
-var PORT = process.env.PORT || 3000;
+// const fs = require('fs');
+// const path = require('path');
+const webpack = require('webpack');
+const WebpackDevServer = require('webpack-dev-server');
+const openInEditor = require('express-open-in-editor');
+const sourceFragment = require('express-source-fragment');
+const config = require('./webpack.config');
+const PORT = process.env.PORT || 3000;
 
-var server = new WebpackDevServer(webpack(config), {
+const options = {
   publicPath: config.output.publicPath,
+  contentBase: '.',
   hot: true,
+  host: 'localhost',
   historyApiFallback: true,
   stats: {
     colors: true
   },
-  setup: function(app) {
+  before(app) {
     app.use('/open-in-editor', openInEditor({
       cwd: __dirname
     }));
     app.use('/source-fragment', sourceFragment({
       cwd: __dirname
     }));
+    /*app.use('/dist', function(req, res) {
```

Comment:
Looks like it should be removed

Reply:
I don't know. Maybe we can uncomment this code and use `component-inspector` from local build for react example instead of using `component-inspector`  from npm module. What do you think about it?

Reply:
You can use `npm link` or make a symlink (`node_modules/component-inspector` -> `../..`) for that. I use `npm link` usually.